ChatAPI.WebAPI/Program.cs

```diff
@@ -41,4 +42,9 @@
 app.MapControllers();
 app.MapHub<ChatHub>("/chatHub");
 
+using (IServiceScope scope = app.Services.CreateScope())
+{
+	scope.ServiceProvider.GetRequiredService<AppDbContext>().ApplyMigrations();
```

Comment:
The only improved version of this which I found is moving this logic in the ConfigureServices method, so that we apply migrations before the code above is run.

Reply:
I am pretty sure it can be done in the db context by some manner